common/src/main/resources/assets/estrogen/lang/de_de.json

```
@@ -73,16 +126,25 @@
   "item.estrogen.estrogen_pill.tooltip": "Östrogenpille",
   "item.estrogen.estrogen_pill.tooltip.summary": "Temporäre Quelle für _Frauenpower_.",
   "item.estrogen.filtrated_horse_urine_bucket": "Filtriertes Pferdeurin-Eimer",
+  "item.estrogen.gender_change_potion": "Trank der Geschlechtsumwandlung",
+  "item.estrogen.gender_fluid_bucket": "Flüssiges Gender-Eimer ",
   "item.estrogen.horse_urine_bottle": "Pferdeurin-Flasche",
   "item.estrogen.horse_urine_bucket": "Pferdeurin-Eimer",
   "item.estrogen.incomplete_estrogen_patches": "Unvollständiges Östrogenpflaster",
   "item.estrogen.incomplete_uwu": "Unvollständiges UwU",
   "item.estrogen.liquid_estrogen_bucket": "Flüssiges Estrogen-Eimer",
   "item.estrogen.molten_amethyst_bucket": "Geschmolzener Amethyst-Eimer",
   "item.estrogen.molten_slime_bucket": "Geschmolzener Schleim-Eimer",
+  "item.estrogen.moth_elytra": "Rosarote Ahornelytra",
+  "item.estrogen.moth_fuzz": "Mottenflaum",
+  "item.estrogen.moth_spawn_egg": "Rosarote Ahornmotten-Spawn-Ei",
   "item.estrogen.testosterone_chunk": "Testosteronklumpen",
   "item.estrogen.testosterone_mixture_bucket": "Testosteronmixtur-Eimer",
   "item.estrogen.testosterone_powder": "Testosteronpulver",
+  "item.estrogen.thigh_highs": "Oberschenkelstrümpfe",
+  "item.estrogen.thigh_highs.desc": "§7§oVerfärbar",
+  "item.estrogen.thigh_highs.tooltip": "Oberschenkelstrümpfe",
+  "item.estrogen.thigh_highs.tooltip.summary": "Ein paar Oberschenkelstrümpfe das dir _Fallschadenresistenz_ gibt. Normale können _verfäbrt_ werden, mit 2 Farbstoffen auf beiden Seite in einer _Werkbank_.",
```

```suggestion
  "item.estrogen.thigh_highs.tooltip.summary": "Ein Paar Oberschenkelstrümpfe das dir _Fallschadenresistenz_ gibt. Normale können _verfärbt_ werden, mit 2 Farbstoffen auf beiden Seite in einer _Werkbank_.",
```
It's one unit of exactly two, "a pair", so the (capitalized) noun "Paar" should be used here.

In English, "ein Paar" would be "a pair [of socks]" or "a couple" as in two people marrying; "ein paar" would be like "a few" or "a couple [of apples]".

(Also small typo)